cmd/triggerrun/cmd/root.go

```diff
@@ -208,11 +212,11 @@ func processTriggerSpec(kubeClient kubernetes.Interface, client triggersclientse
 	if tri.Namespace == "" {
 		tri.Namespace = "default"
 	}
-
+	client.TriggersV1alpha1().TriggerBindings(tri.Namespace).List(context.Background(), metav1.ListOptions{})
```

Comment:
hmm, looks like we aren't really doing anything with this line 😛  remove?

Reply:
It's interesting that if I don't make any call directly using triggerClient(like Line#215), the Listers can't get any existing TriggerBinding or TriggerTemplate from the namespace. I'm new to Kubernetes but I guess the client does some preparation when called. I use about one day to find out this, but I can't figure out why Lister doesn't work without this line.

Reply:
Besides, it only happens here. The Listers in cmd/eventlistenersink/main.go are totally fine.

Reply:
that interesting, maybe some kind of initialization that's missing? But one thing I just realized is that for cmd/triggerrun, it's ok to just make the direct API calls instead of using the lister since its a CLI tool. It'll only make the call once when the CLI tool is executed so using a Lister cache isn't all that useful.
I think passing in the direct triggersClient instead of the lister should be fine for this usage.

Reply:
That makes sense. I will go back to the old way.